code/nrf-connect/prstlib/include/prstlib/led.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PRST_LED_H_
+#define _PRST_LED_H_
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/logging/log.h>
```

Comment:
`#include <zephyr/kernel.h>` seems to be needed here

Reply:
Done, tks.